modules/openideal_slideshow/templates/slideshow.html.twig

```diff
@@ -1,5 +1,77 @@
+{# The swiper slideshow. #}
```

Comment:
1) I think we can remove the following line from `openideal_slideshow_theme()`:
`'template' => 'slideshow',`

In that case, template will have default name - `openideal-slideshow.html.twig`.

2) I don't see how do we handle cache metadata of the slideshow images. Please make sure the build array has appropriate cache tags/contexts.